src/Discord/Internal/Types/ScheduledEvents.hs

```diff
@@ -0,0 +1,188 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE RecordWildCards #-}
+
+-- | Structures pertaining to Discord Scheduled Events
+module Discord.Internal.Types.ScheduledEvents where
+
+import           Data.Aeson                     ( (.=)
+                                                , ToJSON(toJSON)
+                                                , Value(Null, Number, Object)
+                                                , object
+                                                )
+import           Data.HashMap.Strict            ( insert )
+import           Data.Maybe                     ( isJust )
+import qualified Data.Text                     as T
+import           Data.Time                      ( UTCTime )
+import           Discord.Internal.Types.Prelude ( ChannelId
+                                                , GuildId
+                                                , Snowflake
+                                                , UserId
+                                                )
+import           Discord.Internal.Types.User    ( User )
```

Comment:
This is brittany formatting for module imports? Intriguing.

Reply:
Yup that's brittany